src/pages/Home/index.js

```diff
@@ -72,46 +74,157 @@ function Deals({ item }) {
   );
 }
 
+function ProductModal({ selectedId, closeModal, product }) {
+  const [cart, setCarting] = useState([]);
```

Comment:
```suggestion
  const [cart, setCart] = useState([]);
```

The state and the setter should be named the same

Reply:
Very true